src/components/assets/AnchorLinks.vue

```diff
@@ -43,7 +63,20 @@ export default defineComponent({
       section?.scrollIntoView({ behavior: 'smooth', block: 'start' });
     };
 
-    return { nativeTokenSymbol, scrollTo };
+    const { rewards } = useDappStaking();
+
+    const isStakerRewards = computed<boolean>(() => {
```

Comment:
Those variables are already defined in `useDappStaking`. Please reuse them
<img width="228" alt="image" src="https://github.com/AstarNetwork/astar-apps/assets/8452361/bbe0c64c-b136-45aa-83d2-05ccd7ed2994">

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1193/commits/6d9e6d81d4cd3119e64d8a2bdec68caa45594cc0) commit.